src/utils/obsidian/map-idx-to-cursor-position/map-idx-to-cursor-position.ts

```diff
@@ -0,0 +1,31 @@
+import { EditorPosition } from "obsidian";
+import { NEWLINE_SYMBOL } from "../../constants";
+
+export function mapIdxToCursorPosition(
```

Comment:
The opposite function for `mapCursorPositionToIdx` dedicated to transforming selection indices into an Obsidian format